acceptance/go.mod

```diff
@@ -82,12 +82,12 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/urfave/cli v1.22.2 // indirect
 	go.uber.org/atomic v1.7.0 // indirect
-	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83 // indirect
-	golang.org/x/net v0.0.0-20211216030914-fe4d6282115f // indirect
+	golang.org/x/crypto v0.1.0 // indirect
```

Comment:
I got a warning in my IDE this had an outstanding CVE

Reply:
I wonder if we should just explicitly define a newer version of that package to be safe. The latest version is 0.8.0 for that package. Not sure what the correct choice is here, considering it's an indirect dependency.